Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/OutboundDocRetrieveAuditTransformer_a0.java

```diff
@@ -30,6 +30,7 @@
 import gov.hhs.fha.nhinc.common.auditlog.DocRetrieveMessageType;
```

Comment:
I just looked at this class and did a usage search after looking at the workflow.  I don't think this class is used now.  It can be deleted along with it's unit test.